Paragraphs/Add-text-box/.NET-Standard/Add-text-box/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+using System.IO;
+
+namespace Add_text_box
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Creates a new Word document.
+            using (WordDocument document = new WordDocument())
+            {
+                //Adds new section to the document.
+                IWSection section = document.AddSection();
+                //Adds new paragraph to the section.
+                IWParagraph paragraph = section.AddParagraph();
+                //Appends new textbox to the paragraph.
+                IWTextBox textbox = paragraph.AppendTextBox(150, 75);
+                textbox.TextBoxFormat.Height = 115;
```

Comment:
Optimize this code and set necessary size in appendtextbox API itself.

Reply:
Optimised the code as per suggested.